Cli patch 20 11 20 (#392)
Agoric/documentation: main/getting-started/agoric-cli-guide.md

After using `agoric start` to test with the simulated chain, stop the simulated
chain with Control-C.

You must first compile the Golang dependencies in the Agoric SDK by running
```sh
cd agoric-sdk/packages/cosmic-swingset && make
```
Check that the `ag-cosmos-helper` binary is in your `$PATH` and works by running the following. If not, you
need to add it to your `$PATH`:
```sh

> ag-cosmos-helper version --long

Reviewer: ```suggestion
# Display the directory that should be in your $PATH.
echo ${GOBIN-${GOPATH-$HOME/go}/bin}
# Attempt to run a binary that was installed there.
ag-cosmos-helper version --long
```

```sh
agoric start local-solo <PORT-NUMBER>
```
For each new local ag-solo, you need to either:
- Open the wallet corresponding to each local-solo port number by running:
  - `agoric open --hostport=localhost:<local-solo-portnumber>`
- Or open the wallet, get its URL displayed, and then copy/paste the URL into the browser you want to
  view the wallet in:
  - `agoric open --no-browser --hostport=localhost:<local-solo-portnumber>` 

Test that each of your ag-solos is running and has a wallet by connecting to `http://localhost:<PORT-NUMBER>/` for each port you used.

> Run your contract and API deployment scripts against the API ag-solo with this `agorice deploy` command. Note that port 8000 is the default 

Reviewer: ```suggestion
Run your contract and API deployment scripts against the API ag-solo with this `agoric deploy` command. Note that port 8000 is the default 
```

![Agoric Registration](./assets/Agoric-Registration.png)

Keep the recommended address (`http://localhost:8000`), click **Open**, and verify that it opens a wallet and 
REPL (Read-Eval-Print-Loop).  

In the REPL, type `console.log(8000)`, hit **Enter**, and see the 
command and output in the REPL history.

![REPL](./assets/REPL-1.png)

#### Configure an additional client browser
Open a different browser. Not just another tab or window, but a completely different browser or browser profile 
that doesn’t share cookies with any other ag-solo’s browser. For example, if you had an open Chrome window, 
create a new Chrome profile, or open a Firefox or Safari window. 

Navigate to `https://local.agoric.com/` and, 
in the resulting page, set the address to `http://localhost:8001` 

Click **Save**. Then click **Open** and 

> verify that it opens a different wallet and REPL (the REPL history will be blank). 

Reviewer: ```suggestion
verify that it opens a wallet page (but note that you won't have access yet).

You will need to run `agoric open --repl --hostport=localhost:8001` to actually connect to the wallet and REPL (the REPL history will be blank). 
```
